script.py

```diff
@@ -0,0 +1,114 @@
+import os
+# # SRE-basic-stack-Ubuntu-21.04
+# CLI fore clear ubuntu desktop v21.04
+print('This is a script for sre installation')
+
+
+print('********************************************')
+# ## Directories
+print("First we are going to create the directories")
+# ```
+# mkdir ~/Studies
+# mkdir ~/Work
+# ```
+dir1 = 'mkdir ~/Studies'
+dir2 = 'mkdir ~/Work'
+
+os.system(dir1)
+os.system(dir2)
+print('********************************************')
+
+# ## Req
+# ```
+# sudo apt-get update
+# ```
+
+apt = 'sudo apt-get update'
+os.system(apt)
+
+print('********************************************')
+# ## Git config
+# ```
+# sudo apt-get install git-all
+# git config --global user.name "yabouseada"
+# git config --global user.email "\gitEmail"
+# ```
+gitName = input("Enter Your Github Name:  ")
```

Comment:
create a method for user configuration with the input part as the first part of the code